texspell.sh

```diff
@@ -603,6 +603,85 @@ function tex_parser_opendetex {
 
 }
 
+##################
+# Type checker #
+##################
+# 1 - Path to a plaintex file to correct
+# 2 - Path to a tmp file with
+#   * Line of error
+#   * Offset and range of the error
+#   * type of error + proposition to correct
+function type_checker_hunspell {
```

Comment:
Maybe describe a bit more the role of the function
and/or
is `type_checker_hunspell` an appropriate name ? What it does is *spell check*, not *type*, right ?